infoschema/builder.go

```diff
@@ -166,6 +162,9 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 				}
 				continue
 			}
+			// Reduce the impact on DML when executing partition DDL. eg.
+			// While session 1 performs the DML operation associated with partition 1,
+			// the TRUNCATE operation of session 2 on partition 2 does not cause the operation of session 1 to fail.
```

Comment:
Why move the comment here? It intends to explain why to update `tblIDs` for `ActionTruncateTablePartition`.

Reply:
I wanted to move it to the line after `ActionTruncateTablePartition`, a mistake.